docs/index.rst

```diff
@@ -6,16 +6,27 @@ HDFS Connector
 The HDFS connector allows you to export data from Kafka topics to HDFS files in a variety of formats
 and integrates with Hive to make data immediately available for querying with HiveQL.
 
+The connector periodically polls data from Kafka and writes them to HDFS. The data from each Kafka
+topic is partitioned by the provided partitioner and divided into chucks. Each chunk of data is
+represented as an HDFS file with topic, kafka partition, start and end offset of this data chuck
+in the filename. If no partitioner is specified in the configuration, the default partitioner which
+preserves the Kafka partitioning will be used. The size of each data chunk is determined by the
+number of records written to HDFS and the time written to HDFS.
+
+The HDFS connector integrates with Hive and when it is enabled, the connector automatically creates
+an external Hive partitioned table for each Kafka topic and updates the table according to the data
+available in HDFS.
+
 Quickstart
 ----------
 In this Quickstart, we use the HDFS connector to export data produced by the Avro console producer
 to HDFS.
 
 Start Zookeeper, Kafka and SchemaRegistry is you haven't done so. The instructions on how to start
-these services is available in this document. You also need to have Hadoop running locally or
-remotely and makes sure that you know the HDFS url. For Hive integration, you also need to have
-Hive installed. This Quickstart assumes that you started the required services with the default
-config and you should make necessary changes according to the actual configuration used.
+these services is available at the Confluent Platform QuickStart. You also need to have Hadoop
+running locally or remotely and makes sure that you know the HDFS url. For Hive integration, you
```

Comment:
only if the connector has privileges to do it. In many Hadoop clusters, it will not have write access to "/"...